database/factories/AuthorBookFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\AuthorBook;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class AuthorBookFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = AuthorBook::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            //
```

Comment:
You should finish this method implementation.  
Same for other factory classes.

Reply:
Done.